WordPress/Classes/Extensions/MediaLibrary+Local.swift

```diff
@@ -83,10 +87,10 @@ extension MediaService {
 
     /// Calculates the allocated size of the Media directory, in bytes, or nil if an error was thrown.
     ///
```

Comment:
Maybe we should add the documentation for the block parameter and explain that the return value is the size in bytes?